solutions/src/test/java/ru/sberbank/school/task13/beanfieldcopier/util/MyUser.java

```diff
@@ -0,0 +1,34 @@
+package ru.sberbank.school.task13.beanfieldcopier.util;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Objects;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class MyUser {
+    protected String name;
+    protected Integer age;
+    protected boolean ready;
+    protected MyUser parent;
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a110265457>*